packages/backend-rest-api/README.md

```diff
@@ -0,0 +1,44 @@
+# @aws-amplify/backend-rest-api
+
+This package provides REST API support for AWS Amplify Gen2 applications.
+
+## Installation
+
+```bash
+npm install @aws-amplify/backend-rest-api
+```
+
+## Usage
+
+```typescript
+import { defineBackend } from '@aws-amplify/backend';
+import { defineRestApi } from '@aws-amplify/backend-rest-api';
+
+const api = defineRestApi({
```

Comment:
Can you add the following?
1. use Amplify Auth (aka `defineAuth`) to authorize requests?
2. Make sure that Rest API url is in included in amplify outputs file? (aka client config)